src/infra/monitoring/grafana/Dockerfile

```diff
@@ -1,34 +1,32 @@
-# Use a standard node image as the build container
+# Even though we're not building but only installing, we're still doing multi-stage because grafana does not have NPM
 FROM node:16 as builder
 
-# Copy the files into the builder container
-WORKDIR /usr/src/app
+# Copy all source files over to our working directory
+WORKDIR /app
 COPY . .
 
-# Build the healthmodel panel project
-WORKDIR /usr/src/app/healthmodelpanel
-RUN ["yarn", "install"]
-RUN ["yarn", "build"]
-
-# We also need to edit the dashboard file and insert the queries from separate files
-# We have a separate script to do this, called insertqueries.sh
-WORKDIR /usr/src/app/config
+# The queries are stored separately from the dashboard definitions, for maintainability reasons. 
+# The insertqueries script replaces the placeholders in the dashboard files with the actual KQL queries 
+WORKDIR /app/config
 RUN chmod +x insertqueries.sh && ./insertqueries.sh
 
-# Build phase is done, now build the container. Start with the office Grafana image
-# We're using 8.3.4 for now since in later versions our frontdoor origin is not accepted. 
-# This is a known issue and we'll upgrade to a later version once that's resolved. 
+# The healthmodel panel is a third party component, published on NPM. 
+# It contains only the built solution, so there is nothing to build, only install. 
+WORKDIR /app/healthmodelpanel
```

Comment:
so where does the healthmodel gets pulled in?

Reply:
Line 16. Based on what's put in package.json.

Reply:
ok, maybe just add a comment when this working folder is needed